dashboard/app/controllers/pd/workshop_daily_survey_controller.rb

```diff
@@ -225,6 +225,67 @@ def new_post
       render :new_general
     end
 
+    # GET /pd/workshop_survey/csf/:survey_name/:enrollment_code
```

Comment:
Since this action is handling specifically the pre-and post workshop surveys for deep dive and not all csf surveys (bc not all are in jotform yet), can you add a comment like on the `new_general` and `new_facilitator` actions to explain it?

Reply:
Will do